tsa-core/src/main/kotlin/org/usvm/machine/types/TvmDataCellTypesHandlers.kt

```diff
@@ -158,6 +185,55 @@ fun <ReadResult : TvmCellDataTypeReadValue> TvmStepScopeManager.makeSliceTypeLoa
     )
 }
 
+private fun <ReadResult : TvmCellDataTypeReadValue> retryWithBitvectorRead(
+    type: TvmCellDataIntegerRead,
+    load: TvmDataCellLoadedTypeInfo.LoadData<ReadResult>,
+    tlbStack: TlbStack,
+    state: TvmState,
+    stepResult: TlbStack.Error,
+    manager: TvmStepScopeManager,
```

Comment:
Please, call this scope. (If this name is used in other places as well, rename it)